src/admin/tmpl/plugins/default.php

```diff
@@ -64,21 +66,39 @@
                     </thead>
                     <tbody>
                         <?php foreach ($this->items as $i => $item) :
-                            $link = Route::_('index.php?option=com_plugins&client_id=0&task=plugin.edit&extension_id=' . $item->extension_id . '&tmpl=component&layout=modal');
-                            $linkModal = HTMLHelper::_(
-                                'link',
-                                '#plugin' . $item->extension_id . 'Modal',
-                                $item->name,
-                                'data-bs-toggle="modal" id="title-' . $item->extension_id . '"'
-                            );
-
-                            if (!empty($item->note)) {
-                                $linkModal .= "<div class=\"small\">" . Text::sprintf('JGLOBAL_LIST_NOTE', $this->escape($item->note)) . "</div>";
-                            }
-
                             $canEdit    = $user->authorise('core.edit', 'com_plugins');
                             $canCheckin = $user->authorise('core.manage', 'com_checkin') || $item->checked_out == $user->id || is_null($item->checked_out);
                             $canChange  = $user->authorise('core.edit.state', 'com_plugins') && $canCheckin;
+                            $link = Route::_('index.php?option=com_plugins&client_id=0&task=plugin.edit&extension_id=' . $item->extension_id . '&tmpl=component&layout=modal', false);
+                            if (version_compare($joomlaVersion->getShortVersion(), '5.1', '<')) {
+                                $linkModal = HTMLHelper::_(
+                                    'link',
+                                    '#plugin' . $item->extension_id . 'Modal',
+                                    $item->name,
+                                    'data-bs-toggle="modal" id="title-' . $item->extension_id . '"'
+                                );
+                            } else {
+                                $popupOptions = [
+                                    'popupType'  => 'iframe',
+                                    'textHeader' => Text::sprintf('COM_OCHSUBSCRIPTIONS_CPANEL_MANAGER_PLUGIN', $item->name),
```

Comment:
you have introduced a language key string from OCHSUBSRIPTIONS

Reply:
my bad, fixed!